src/components/Calender.vue

```diff
@@ -97,44 +106,47 @@
 		transition: background-color 1.6s ease-in;
 	}
 
-	a.date.padding
-	{
+	a.date.padding {
 		opacity: 0;
 		pointer-events: none;
 	}
 
 	a.date:hover,
-	a.date:focus
-	{
+	a.date:focus {
 		background-color: var(--red);
 		transition: background-color 0.3s ease;
 	}
-	
-	h3
-	{
+
+	h3 {
 		font-family: var(--numbers-font);
 		font-size: 2.5rem;
 		font-weight: normal;
 
-		padding: 0.22em 0.11em 0;
 		margin: 0;
 		margin-bottom: auto;
-		border-radius: 100%;
+		text-align: left;
+	}
 
+	.number {
+		padding: 0.22em 0.11em 0;
+		margin: 0 0 auto;
+
+		border-radius: 100%;
 		line-height: 2ch;
 		width: 2ch;
-		text-align: center;
+	}
 
-		transform: translate(-5px, -5px);
+	.title {
+		font-size: 1.4rem;
+		padding: 0.22em 0 0 0;
 	}
 
-	.today h3
-	{
+	.today .number {
+		transform: translate(-5px, -5px);
```

Comment:
The translation here was previously applied to all the date numbers, to allow for the current date's red circle to overlap off the square. Applying it to _only_ the current date's number does still achieve that, but aligns the number noticeably different from all the others.